src/structures/ApplicationServer.ts

```diff
@@ -10,6 +10,13 @@ import {
 } from '../common/app';
 import caseConv from '../util/caseConv';
 
+export type ApplicationServerStatus =
+    | 'installing'
+    | 'install_failed'
+    | 'reinstall_failed'
+    | 'suspended'
+    | 'restoring_backup';
```

Comment:
This already exists:

https://github.com/PteroPackages/PteroJS/blob/0823869d7064ca4f15a2611e4aa977950fa204b9/src/common/app.ts#L197-L203

Reply:
Resolved